src/test/resources/org/prebid/server/it/openrtb2/epsilon/alias/test-auction-epsilon-request.json

```diff
@@ -18,7 +18,7 @@
   "ext": {
     "prebid": {
       "aliases": {
-        "conversantAlias": "conversant"
+        "conversant": "epsilon"
       },
```

Comment:
This `aliases` block is redundant. If you wanna be sure, that aliases feature properly works, change it to be like this:
```
      "ext": {
         "conversantAlias": {
           "site_id": "site_id"
         }
       }
      ....
      "aliases": {
         "conversantAlias": "conversant"
       },
```